library/std/src/sys/thread_local/mod.rs

```diff
@@ -91,13 +91,15 @@ mod lazy {
             }
         }
 
-        /// The other methods hand out references while taking &self.
-        /// As such, callers of this method must ensure no `&` and `&mut` are
-        /// available and used at the same time.
+        /// Watch out: unsynchronized internal mutability!
+        ///
+        /// # Safety
+        /// Unsound if called while any `&'static T` is active.
```

Comment:
References may be active upon entry here, but they mustn't be used anymore.
```suggestion
        /// Causes UB if any reference to the value is used after this.
```